collector/mongod/op_latencies.go

```diff
@@ -16,28 +16,22 @@ package mongod
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
-	"strconv"
 )
 
 var (
-	opLatenciesTotal = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Namespace: Namespace,
-		Name:      "op_latencies_latency_total",
-		Help:      "op latencies statistics in microseconds of mongod",
-	}, []string{"type"})
+	opLatenciesHistogram    *prometheus.HistogramVec = nil
```

Comment:
While we removing old mertics it would nice if you add you changes in https://github.com/percona/mongodb_exporter/blob/master/CHANGELOG.md inside **Unreleased** section